### PR TITLE
Bump go-fastly to new v8 major release to add project-id in GCS for logging

### PR DESCRIPTION
**Pull request by bowrocker**
Bump the `go-fastly` version to v8 in support of adding the missing `project-id` field to the GCS provider for logging.

### CODE REVIEW

File: fastly/block_fastly_service_logging_gcs.go [hunk 1]
```diff
@@ -38,6 +38,11 @@ func (h *GCSLoggingServiceAttributeHandler) GetSchema() *schema.Schema {
 			DefaultFunc: schema.EnvDefaultFunc("FASTLY_GCS_ACCOUNT_NAME", ""),
 			Description: "The google account name used to obtain temporary credentials (default none). You may optionally provide this via an environment variable, `FASTLY_GCS_ACCOUNT_NAME`.",
 		},
+		"project_id": {
```
Comment (Integralist) [inline fastly/block_fastly_service_logging_gcs.go]: This new field needs to be placed in alphabetical order (so just before the "secret_key" field). This is what the CI 'lint' job is failing on currently.

**Comment by Integralist** [comment]
To fix the failing "docs" CI job you'll need to run `make generate-docs` locally and then commit/push the changes.

File: fastly/block_fastly_service_logging_gcs.go (hunk 1 above)
Comment (bowrocker) [reply fastly/block_fastly_service_logging_gcs.go]: Thank you, fixed in e2e4b46f2acc808712cb6bb700c99457294d420d.

File: fastly/block_fastly_service_logging_gcs.go [hunk 2]
```diff
@@ -151,6 +156,7 @@ func (h *GCSLoggingServiceAttributeHandler) Create(_ context.Context, d *schema.
 		Name:             gofastly.String(resource["name"].(string)),
 		Path:             gofastly.String(resource["path"].(string)),
 		Period:           gofastly.Int(resource["period"].(int)),
+		ProjectID:        gofastly.String(resource["project-id"].(string)),
```
Comment (Integralist) [inline fastly/block_fastly_service_logging_gcs.go]: I think this is a typo...

```suggestion
		ProjectID:        gofastly.String(resource["project_id"].(string)),
```

**Review by Integralist — CHANGES_REQUESTED** [review]

File: fastly/block_fastly_service_logging_gcs.go (hunk 2 above)
Comment (bowrocker) [reply fastly/block_fastly_service_logging_gcs.go]: Thank you for catching that, fixed in 3e942ee27f88d4abcf4a5b16d924ff58bf0aec84.

**Review by Integralist — APPROVED** [review]

**Pull request merged**